network-api/networkapi/mozfest/migrations/0018_update_wide_images.py

```diff
@@ -0,0 +1,48 @@
+# # Generated by Django 3.0.14 on 2021-08-10 18:19
+
+from django.db import migrations
+import wagtail.core.blocks
+import wagtail.core.blocks.static_block
+import wagtail.core.fields
+import wagtail.images.blocks
+import wagtailmedia.blocks
+
+
+from networkapi.mozfest.models import MozfestPrimaryPage
+
+
+def loop_through_pages_with_image_blocks(apps, schema):
+
+    for page in MozfestPrimaryPage.objects.all():
```

Comment:
it looks like we'll need an extra check for page aliasses now that we use wagtail-localize:

```suggestion
    for page in MozfestPrimaryPage.objects.all():
        if hasattr(page, 'alias_of') and page.alias_of != None:
            # This is a page alias, rather than a real page
            continue
```

Otherwise we'll run into this problem when we try to apply it to our real data:

```
Running migrations:
  Applying mozfest.0017_auto_20210818_0231... OK
  Applying mozfest.0018_update_wide_images... OK
  Applying mozfest.0019_auto_20210818_0231... OK
  Applying wagtailpages.0031_auto_20210818_0231... OK
  Applying wagtailpages.0032_update_wide_images...Traceback (most recent call last):
  File "network-api/manage.py", line 34, in <module>
    execute_from_command_line(sys.argv)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/base.py", line 85, in wrapped
    res = handle_func(*args, **kwargs)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/commands/migrate.py", line 245, in handle
    fake_initial=fake_initial,
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/db/migrations/executor.py", line 227, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/db/migrations/migration.py", line 124, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/app/network-api/networkapi/wagtailpages/migrations/0032_update_wide_images.py", line 53, in loop_through_pages_with_image_blocks
    update_annotated_image_blocks(page_qs)
  File "/app/network-api/networkapi/wagtailpages/migrations/0032_update_wide_images.py", line 38, in update_annotated_image_blocks
    revision = page.save_revision()
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/wagtail/core/models.py", line 1407, in save_revision
    "save_revision() was called on an alias page. "
RuntimeError: save_revision() was called on an alias page. Revisions are not required for alias pages as they are an exact copy of another page.
1
```